javascript/src/seedsource/containers/Map.jsx

```diff
@@ -21,6 +21,7 @@ import { setPopupLocation, resetPopupLocation } from '../../actions/popup'
 import { setPoint } from '../../actions/point'
 import { isClose } from '../../utils'
 import '../../leaflet-controls'
+import 'leaflet.vectorgrid/dist/Leaflet.VectorGrid.js'
```

Comment:
You should be able to just do

```javascript
import 'leaflet.vectorgrid'
```

It knows how to find the source correctly from the `package.json`.